ReactiveCocoa/Swift/Signal.swift

```diff
@@ -177,6 +177,13 @@ public final class Signal<Value, Error: Swift.Error> {
 	}
 }
 
+// Testability
+extension Signal {
+	internal var test_observerCount: Int? {
```

Comment:
This is a consequence of the simplified binding implementation. As the operators now return directly the disposables from `Signal.observe` or `SignalProducer.startWithSignal`, inspecting through `isDisposed` of those now just means whether it is manually disposed of.

I consider this a middle ground between making everything `internal` for tests, and having no hooks to test for.